introduction.md

```diff
@@ -3,3 +3,10 @@ Introduction
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?